### ISSUE TITLE
Overriding default plyr settings does not merge correctly

### ISSUE THREAD

**Issue by martinbroos** [biati-digital/glightbox#108]
**Describe the bug**
Overriding default plyr settings does not merge correctly

**Are you able to reproduce the bug in the demo site**
No

**To Reproduce**

```
const lightbox = GLightbox({
    plyr: {
        config: {
            youtube: {
                noCookie: false, // when enabled video does not play
            },
        },
    },
});

lightbox.setElements(items);

lightbox.open();
```

**Expected behavior**
I only want to override some plyr settings but this way it will remove other plyr settings which will result in errors. Eg. that plyr lib will not be loaded because the url to it is missing.

**Comment by biati-digital** [comment]
I'm working on an update to fix this 😃

**Comment by biati-digital** [comment]
Fixed in the latest version 2.0.6
